src/fprime_gds/common/pipeline/dictionaries.py

```diff
@@ -121,8 +122,13 @@ def load_dictionaries(self, dictionary, packet_spec):
             msg = f"[ERROR] Dictionary '{dictionary}' does not exist."
             raise Exception(msg)
         # Check for packet specification
-        if packet_spec is not None:
-            packet_loader = fprime_gds.common.loaders.pkt_xml_loader.PktXmlLoader()
+        if self._metadata["dictionary_type"] == "json" and packet_set_name is not None:
+            packet_loader = fprime_gds.common.loaders.pkt_json_loader.PktJsonLoader(dictionary)
```

Comment:
It is my expectation that most topologies (or at least in early development) will have no more than 1 packet set. In that case, it would be nice to pick that one and only packet set by default; and provide a meaningful error message if there are two or more packet sets. Thoughts?

Reply:
Wait I think I may have misunderstood what packet sets are when reviewing this
https://nasa.github.io/fpp/fpp-spec.html#Specifiers_Telemetry-Packet-Set-Specifiers

Should we not import **all** packet sets instead?

Reply:
Topologies can have multiple packet sets. I am thinking that for now, we keep the `packet-set-name` option and _do not_ include a default if there is one packet set in the dictionary since that is most clear and matches what we currently support in the XML. For the future, I think we should add an ID to the packet set name in FPP and downlink the packet set ID and packet ID so the GDS can decode packets from any packet set.

Reply:
Sounds good, thanks for clarifying.